example/terraform/main.tf

```diff
@@ -5,12 +5,26 @@ provider "layer0" {
 }
 
 module "consul" {
-  source         = "github.com/quintilesims/consul/terraform"
-  environment_id = "${layer0_environment.dev.id}"
+  source         = "../../terraform"
+  environment_id = "${layer0_environment.consul.id}"
+}
+
+resource "layer0_environment" "consul" {
```

Comment:
Just want to make a future note that adding the consul environment in this module will likely have implications for the layer0 walkthroughs that consume this module (for instance, if the walkthroughs have terraform files that make their own consul environment)

Reply:
This environment is created by the HelloWorld application, I kept the Consul module behavior (of not creating an environment by itself) as-is.